src/Components/Link/Link.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '@material-ui/core/Button';
+
+const LinkButton = ({ active = false, children, onClick }) => {
+    if (active) {
+        return (
+            <span
+                style={{
+                    marginLeft: '6px',
```

Comment:
Is it correct? I think It should be place in a object.

Reply:
No, it's not correct. I already fixed this using the makeStyle function and adding classes to Component